src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -113,7 +117,7 @@ public HetPulldownCalculator(final File refFile, final File snpFile,
      * </p>
      * @param baseCounts        map of base-pair counts
      * @param totalBaseCount    total base-pair counts (excluding N, etc.)
-     * @param pvalThreshold     p-value threshold for two-sided binomial test
+     * @param pvalThreshold     p-value threshold for two-sided binomial test (should be in [0, 1], but no check is performed)
```

Comment:
it seemed a bit redundant to check for every het, since it's checked in the true public entry point in the getNormal method.  i'll change this method to VisibleForTesting instead.